docker/opensearch/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM opensearchproject/opensearch
+
+USER root
+RUN yum install -y python3 pip3 util-linux
+RUN useradd -ms /bin/bash test
+
+# Replace the provided OpenSearch config file
+USER opensearch
+ADD --chown=opensearch config/opensearch.yml /usr/share/opensearch/config/opensearch.yml
```

Comment:
Can "config/opensearch.yml" be parametrized with environment variable?

Reply:
It can. Another option is [Docker ARGs](https://docs.docker.com/engine/reference/builder/#arg) which may be more suitable in this case, since they are more of a build time tool where as `ENV` variables persist in the container runtime. What do you think?

Reply:
Refer to above comment. Can we do in docker compose?